deploy.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Deploy WP site
+  hosts: web
+  remote_user: deploy
```

Comment:
Yeah good catch. Realized last night we may want to rename `deploy_user` to something a little more generic like `web_user`. Because in development there is no "deploy" and it's just the `vagrant` user.